lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -350,9 +351,11 @@ def _create_node_capabilities(
         node_capabilities.network_interface.nic_count = 1
         node_capabilities.gpu_count = 0
         security_profile_setting = SecurityProfileSettings()
+        cvm_nested_virtualization = CVMNestedVirtualization.create_setting()
         node_capabilities.features = search_space.SetSpace[schema.FeatureSettings](
             is_allow_set=True,
             items=[
+                cvm_nested_virtualization,
```

Comment:
Can we add this feature only when the guest is a CVM guest?

Reply:
The node is configured after this step, so I don't think a check can be added at this exact point. This feature simply acts as a flag for the test to be enabled, functionally it doesn't do anything. But since this feature is being supported, so I think it should be added to the node capabilities.

Reply:
> Can we add this feature only when the guest is a CVM guest?

What means "guest" here? If it means the vm image, it cannot be checked here. If it means the host supports CVM or not, it should be set here. If it's not easy to know the host support CVM or not, a switch can be added in schema to control from the runbook.

Reply:
>  If it means the vm image

Yeah, I mean the VM that is being created by this platform.

> If it means the host supports CVM or not, it should be set here.

I guess I was a bit confused about what the feature `CVMNestedVirtualization` indicates. Based on the name I thought it is used to indicate if a node is a nested CVM.

But if it is used to indicate whether a host supports CVM or not, then it shouldn't be added as a requirement for the `verify_nested_cvm_attestation_report` test case in the first place. @smit-gardhariya ?

Reply:
A feature means the capability of the platform, what kind of VM is capable to create. It doesn't consider if the *image* is CVM or not, but only consider if the host is capable to create a CVM.

Reply:
> A feature means the capability of the platform

Platform or node? At the end of the day, a feature is a property of a node right? We access it with `node.features[FeatureName]`.

Reply:
1. The feature is to abstract the operation of a platform on nodes. It goes with nodes, so you can use features on nodes in test cases.
2. CVM comes with two side: 1) the image supports CVM, 2) the virtualization stack supports CVM. The feature is to tell the supportability of the virtualization stack, not the image. In LISA, "capability" means the platform is capable or not, "requirement" means the test case needs something.

Reply:
Thanks for clarifying! In that case, we don't need this PR. The testcase requirement is incorrect. We will fix that.